bindgen/private/bindgen.bzl

```diff
@@ -201,13 +201,13 @@ def _rust_bindgen_impl(ctx):
 
     cc_toolchain, feature_configuration = find_cc_toolchain(ctx = ctx)
 
-    tools = depset([clang_bin], transitive = [cc_toolchain.all_files])
+    tools = depset(([clang_bin] if clang_bin else []), transitive = [cc_toolchain.all_files])
 
     # libclang should only have 1 output file
     libclang_dir = _get_libs_for_static_executable(libclang).to_list()[0].dirname
 
     env = {
-        "CLANG_PATH": clang_bin.path,
+        "CLANG_PATH": (clang_bin.path if clang_bin else "/dev/null"),
```

Comment:
Does this still work if we just omit the CLANG_PATH key in this case? (If yes, plz do that as it a bit cleaner)

Reply:
Done.